src/styles/stars.scss

```diff
@@ -0,0 +1,31 @@
+.card__container {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin-top: 16px;
+
+  &-reviews {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $main-accent-color;
+  }
+
+  .stars {
+    display: flex;
+
+    &__star {
+      width: 16px;
+      height: 16px;
+      margin-right: 4px;
+
+      background-image: url(../images/star.svg);
+      background-repeat: no-repeat;
+      background-position: center;
+
+      &:nth-child(-n + 4) {
```

Comment:
Not fixed:

![image](https://github.com/user-attachments/assets/70c8c97b-28ed-43a2-8280-5477f1edc0da)

You should add selector to all possible numbers of active stars (from 0 to 5)